cloudant-sync-datastore-core/src/main/java/com/cloudant/mazha/CouchClient.java

```diff
@@ -409,6 +409,9 @@ public void putAttachmentStream(String id, String rev, String attachmentName, St
 
         Map<String, Object> options = new HashMap<String, Object>();
         options.put("revs", true);
+        // by adding latest we should never receive a "missing" response from the server. An ancestor
```

Comment:
I don't think this comment is quite right. The behaviour is that the latest descendant will be returned even if the ancestor we asked for has been deleted.